### ISSUE TITLE
Ejabberd does not start due to error (bad_type, ejabberd_commands)

### ISSUE THREAD

**Issue by alexara** [processone/ejabberd#1291]
> What version of ejabberd are you using?

Master (https://github.com/processone/ejabberd/commit/d0761039ff9b69152854a90fb7a74956e4dff77c)

> What operating system (version) are you using?

Debian 8

> How did you install ejabberd (source, package, distribution)?

Source; copied old configuration and databases from the former 16.06 installation

> What did not work as expected? Are there error messages in the log? What
> was the unexpected behavior? What was the expected result?

Ejabberd does not start with the following error message:

```
Erlang/OTP 17 [erts-6.2] [source] [smp:4:4] [async-threads:10] [kernel-poll:true]

Eshell V6.2  (abort with ^G)
(ejabberd@localhost)1> 19:52:55.161 [info] Application lager started on node ejabberd@localhost
19:52:55.253 [info] Application crypto started on node ejabberd@localhost
19:52:55.345 [info] Application sasl started on node ejabberd@localhost
19:52:55.532 [info] Application asn1 started on node ejabberd@localhost
19:52:55.535 [info] Application public_key started on node ejabberd@localhost
19:52:55.653 [info] Application ssl started on node ejabberd@localhost
19:52:55.702 [info] Application fast_yaml started on node ejabberd@localhost
19:52:55.789 [info] Application fast_tls started on node ejabberd@localhost
19:52:55.844 [info] Application fast_xml started on node ejabberd@localhost
19:52:55.880 [info] Application stringprep started on node ejabberd@localhost
19:52:55.918 [info] Application cache_tab started on node ejabberd@localhost
19:52:57.234 [info] Application mnesia started on node ejabberd@localhost
19:53:03.934 [error] CRASH REPORT Process <0.39.0> with 0 neighbours exited with reason: {aborted,{bad_type,{ejabberd_commands,gen_html_doc_for_commands,[documentation],"Generates html documentation for ejabberd_commands",[],0,ejabberd_commands_doc,generate_html_output,[{file,binary},{regexp,binary},{examples,binary}],restricted,[],{res,rescode},["Path to file where generated documentation should be stored","Regexp matching names of commands or modules that will be included inside generated document","Comma separated list of languages (choosen from java, perl, xmlrpc, json)that ..."],...}}} in application_master:init/4 line 133
19:53:03.937 [info] Application ejabberd exited with reason: {aborted,{bad_type,{ejabberd_commands,gen_html_doc_for_commands,[documentation],"Generates html documentation for ejabberd_commands",[],0,ejabberd_commands_doc,generate_html_output,[{file,binary},{regexp,binary},{examples,binary}],restricted,[],{res,rescode},["Path to file where generated documentation should be stored","Regexp matching names of commands or modules that will be included inside generated document","Comma separated list of languages (choosen from java, perl, xmlrpc, json)that ..."],...}}}
```

**Comment by badlop** [comment]
A similar report was resolved by simply reinstalling ejabberd: #1248. Try installing it from zero, check it starts. Then make your changes in the config one by one, start and check it starts, etc. This way, whatever breaks it, you will know easily what it is.

**Comment by alexara** [comment]
I could resolve the issue by copying all  DB files but `schema.DAT`.
